book.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] Kindly make sure that the `rentals` is accessible. you can add it to the `attr_accessor` instead